journals/apps/journals/management/commands/publish_journals.py

```diff
@@ -154,16 +160,45 @@ def _handle_delete(self, options):
 
         return journal
 
+    def _handle_update(self, options):
+        """handle update of Journal name and status"""
+        uuid = options['update']
+        publish = options['publish']
+
+        try:
+            journal = Journal.objects.get(uuid=uuid)
+        except Journal.DoesNotExist:
+            raise CommandError('The journal with uuid=%s could not be found', uuid)
+
+        configuration = journal.organization.site.siteconfiguration
+
+        discovery_data = {
+            "status": "active" if publish else "inactive",
+            "title": journal.name
+        }
+        if update_service(configuration.discovery_journal_api_client, journal.uuid, discovery_data, "discovery"):
+            self.stdout.write("Successfully updated discovery service for Journal UUID {}".format(journal.uuid))
+        else:
+            self.stderr.write("Error in updating discovery service for Journal UUID {}".format(journal.uuid))
+
+        ecommerce_data = {'title': journal.name}
+        if update_service(configuration.ecommerce_journal_api_client, journal.uuid, ecommerce_data, 'ecommerce'):
+            self.stdout.write("Successfully Updated ecommerce service for Journal UUID {}".format(journal.uuid))
+        else:
+            self.stderr.write("Error in updating ecommerce service for Journal UUID {}".format(journal.uuid))
+
+        return journal
+
     def handle(self, *args, **options):
         """ publish journal info to required services """
 
         if options['create']:
             journal = self._handle_create(options)
-            self.stdout.write(self.style.SUCCESS(
-                'Successfully created Journal {} uuid={}').format(journal.name, journal.uuid))
+            self.stdout.write('Successfully created Journal {} uuid={}'.format(journal.name, journal.uuid))
```

Comment:
nit: It should handle unicode strings in output.

Reply:
just tested, `format()` is handling automatically for us
![image](https://user-images.githubusercontent.com/29247391/43646679-f75a086c-974e-11e8-93e9-190c50722447.png)